regression_tests/operator.go

```diff
@@ -0,0 +1,251 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+	"time"
+
+	walletclient "github.com/bitcoin-sv/spv-wallet-go-client"
+	"github.com/bitcoin-sv/spv-wallet-go-client/xpriv"
+)
+
+const (
+	domainLocalHost     = "localhost:3003"
+	adminXPriv          = "xprv9s21ZrQH143K3CbJXirfrtpLvhT3Vgusdo8coBritQ3rcS7Jy7sxWhatuxG5h2y1Cqj8FKmPp69536gmjYRpfga2MJdsGyBsnB12E19CESK"
+	adminXPub           = "xpub661MyMwAqRbcFgfmdkPgE2m5UjHXu9dj124DbaGLSjaqVESTWfCD4VuNmEbVPkbYLCkykwVZvmA8Pbf8884TQr1FgdG2nPoHR8aB36YdDQh"
+	leaderPaymailAlias  = "leader"
+	minimalBalance      = 100
+	defaultGoClientPath = "../../spv-wallet-go-client/regression_tests"
+	defaultJSClientPath = "../../spv-wallet-js-client/regression_tests"
+)
+
+func main() {
+	loadConfigFlag := flag.Bool("l", false, "Load configuration from .env.config file")
+	flag.Parse()
+
+	var config *Config
+	var err error
+	user := &User{}
+
+	if *loadConfigFlag {
+		config, err = LoadConfig()
+		if err != nil {
+			fmt.Println("Error loading config:", err)
+			return
+		}
+		user.XPriv = config.ClientOneLeaderXPriv
+	} else {
+		config = &Config{}
+	}
+
+	if !IsSPVWalletRunning(domainLocalHost) {
+		fmt.Println("spv-wallet is not running. Run spv-wallet and try again")
+		return
+	}
+
+	sharedConfig, err := GetSharedConfig(adminXPub)
+	if err != nil {
+		fmt.Println("Error getting shared config:", err)
+		return
+	}
+	config.ClientOneURL = sharedConfig.PaymailDomains[0]
```

Comment:
it looks like a repetitive code a bit, maybe consider moving it to separated method? (creating client/operator one and two)

Reply:
moved to separate method